nbodykit/tests/test_lab.py

```diff
@@ -53,3 +53,20 @@ def filter(k, v):
 
     source.save(output="./test_paint-real-%d.bigfile" % comm.size, mode='real')
     source.save(output="./test_paint-complex-%d.bigfile" % comm.size, mode='complex')
+
+@MPITest([1, 4])
+def test_set_options(comm):
+
+    CurrentMPIComm.set(comm)
+
+    with set_options(dask_cache_size=5e9, dask_chunk_size=75):
```

Comment:
Can I use set_options not as a context manager? The name doesn't suggest it is a context manager at all.

Reply:
If it cannot be used without as a context manager, then it can be renamed to `option_context`.
```
with nbkit.option_context(....) as ctx:
    ...
    ...
```

Reply:
Yes it will set the globals if it is not called as a context manager. It largely follows the dask ``set_options`` class.

Reply:
I guess it probably came from numpy's formatting options -- though I am still a bit reserved on their choice of name -- a context named like an action. But looks like we are stuck with such names.